packages/conventional-changelog-core/test/merge-config.spec.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
```

Comment:
the test file names (`merge-config.spec.js` and `index.spec.js`) are based on other packages such as [conventional-commits-parser](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-commits-parser/test), [conventional-changelog-writer](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-writer/test).